### PR TITLE
Change format of nunjucks code and output

### PR DESCRIPTION
**Pull request by yucheng11122017**
…t.md boilerplate

**What is the purpose of this pull request?**

- [X] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**

Currently in [UG](https://markbind.org/userGuide/markBindSyntaxOverview.html#:~:text=Support%20for%20Nunjucks,are%20processed.): 
image

Changed to : 
image

This is to be consistent with other code output formatting in the documentation

**Anything you'd like to highlight/discuss:**
Is there a need to create a new boilerplate just for nunjucks? Current boiler plates will not work with nunjucks due to the need for {%raw%} and {%endraw%} tags to escape the nunjucks. 
Currently only one usage of nunjucks code and output which will be corrected in this PR

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Change format of nunjucks code and output in UG

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [ ] Linked all related issues
- [X] No unrelated changes

### CODE REVIEW

File: docs/userGuide/markBindSyntaxOverview.md [hunk 1]
```diff
@@ -110,15 +113,21 @@ MarkBind has several core Vue.js components built on the popular [BootStrap](htt
 ```
 {% endraw %}
 
-{{ icon_arrow_down }}
+</div>
+
+%%OUTPUT:%%
+<div class="indented">
+
+<box border-left-color={{ markbind_blue }} background-color="white">
```
Comment (lhw-1) [inline docs/userGuide/markBindSyntaxOverview.md:121]: May I know if this part of the `codeAndOutput.md` boilerplate appears blue on your end? The default boilerplate seems to appear grey on my end:

Capture

Or is there a reason why we are changing it to blue specifically?

**Review by lhw-1 — COMMENTED** [review]
Good find & thanks for working on it @yucheng11122017! Seems to work on my end 👍 

> Current boiler plates will not work with nunjucks due to the need for {%raw%} and {%endraw%} tags to escape the nunjucks.

Just to check, you are referring to #2119, right? For now, since this is a viable workaround, maybe we can document this somewhere (or at least mention it in #2119) in case developers run into this exact problem in the future.

File: docs/userGuide/markBindSyntaxOverview.md (hunk 1 above)
Comment (yucheng11122017) [reply docs/userGuide/markBindSyntaxOverview.md]: Apologies, fixed in [479dd7d](https://github.com/MarkBind/markbind/pull/2175/commits/479dd7d548217f8a59a6355ac4e430090d93c9dd). I was working on a PR which changes it to blue

**Comment by yucheng11122017** [comment]
> Good find & thanks for working on it @yucheng11122017! Seems to work on my end 👍
> 
> > Current boiler plates will not work with nunjucks due to the need for {%raw%} and {%endraw%} tags to escape the nunjucks.
> 
> Just to check, you are referring to #2119, right? For now, since this is a viable workaround, maybe we can document this somewhere (or at least mention it in #2119) in case developers run into this exact problem in the future.

Oh yes there are overlaps with #2119. I'm not sure if this would be considered a workaround though since its still not a boilerplate.

**Review by jovyntls — APPROVED** [review]
LGTM, nice that we're improving the consistency of our UG! 

> Is there a need to create a new boilerplate just for nunjucks? Current boiler plates will not work with nunjucks due to the need for {%raw%} and {%endraw%} tags to escape the nunjucks.
Currently only one usage of nunjucks code and output which will be corrected in this PR

IMO If this is the only usage of nunjucks code I think it's fine without a boilerplate :) (Though if we do manage to make the boilerplate work with nunjucks too, that's be great)

**Pull request merged**